src/styles/main.scss

```diff
@@ -1,3 +1,9 @@
-body {
-  margin: 0;
-}
+@import "./utils/variable.scss";
+@import "./utils/mixin.scss";
+@import "./reset.scss";
+
+@import "./blocks/page.scss";
+@import "./blocks/style-block-header.scss";
+@import "./blocks/style-block-nav.scss";
+@import "./blocks/style-block-card.scss";
+@import "./blocks/style-block-stars.scss";
```

Comment:
it's better to avoid repeating 'style-block' in the naming. Consider using 'header', 'nav' instead.